src/MultiFormGoals/ProgressBar/Model.php

```diff
@@ -144,8 +125,9 @@ protected function getDonorCount() {
 	 * @since 2.9.0
 	 **/
 	protected function getDonationCount() {
-		$donations = $this->getDonations();
-		return count( $donations );
+		$query   = new Query( $this->getForms() );
```

Comment:
@henryholtgeerts you'll notice that this is duplicate code, which I plan to refactor in a separate PR.